test/acceptance/features/steps/steps.py

```diff
@@ -425,3 +426,17 @@ def etc_cluster_is_running(context, etcd_name):
         print("etcd cluster not present, creating etcd cluster")
         assert etcd_cluster.create() is True, "etcd cluster is not created"
         assert etcd_cluster.is_present() is True, "etcd cluster is not present"
+
+
+@when(u'Invalid Service Binding is applied')
+def invalid_sbr_is_applied(context):
+    yaml = context.text
+    sbr = ServiceBinding()
+    output = sbr.attempt_to_create(yaml)
+    context.error = output
+
+
+@then(u'Openshift throws an error "{err_msg}"')
```

Comment:
Let's be more generic here in the step definition... since we are actually asserting the content of `context.expected_error` here, the step should rather be something like `Error message "{err_msg}" is thrown` instead of `Openshift throws error...`. 

So please rename the stes to the `Error message "{err_msg}" is thrown` so that it can be re-used by other scenarios. Where the previous steps would set `context.expected_error` to be checked later with this step.

Reply:
Done.

Reply:
PTAL